pkg/client/client.go

```diff
@@ -174,12 +180,40 @@ func printProgress(progress interface{}) error {
 	return nil
 }
 
+// ParseCollectResponse parses a response from a `collect` call
+func ParseCollectResponse(r io.ReadCloser, file io.Writer) (CollectResponse, error) {
+	var resp CollectResponse
+	err := parseGeneric(
+		r,
+		printProgress,
+		func(payload interface{}) error {
+			m, err := base64.StdEncoding.DecodeString(payload.(string))
+			if err != nil {
+				return err
+			}
+
+			_, err = file.Write(m)
+			if err != nil {
+				return err
+			}
+			resp.Exists = true
```

Comment:
Can probably do away with this, since we're setting `resp.Exists` based on what the server tells us in the Result.

Reply:
true.